DeepSkyStacker/StackingDlg.cpp

```diff
@@ -1995,7 +1995,7 @@ void CStackingDlg::retrieveLatestVersionInfo()
 			[this](QNetworkReply* reply) { this->versionInfoReceived(reply); });
 
 		QNetworkRequest req(QUrl("https://github.com/deepskystacker/DSS/raw/release/CurrentVersion.txt"));
-		req.setAttribute(QNetworkRequest::FollowRedirectsAttribute, true);
+		req.setAttribute(QNetworkRequest::RedirectPolicyAttribute, true);
```

Comment:
`QNetworkRequest::FollowRedirectsAttribute` is deprecated, I believe `QNetworkRequest::RedirectPolicyAttribute` is just another name for the same thing.